### ISSUE TITLE
1.39.1 Got invalid value, return 500 error

### ISSUE THREAD

**Issue by love4taylor** [umami-software/umami#1584]
```
Oct 25 12:29:46 localhost nvm-exec[1498080]: PrismaClientValidationError: Argument id: Got invalid value '14ccac36-ce69-4d7e-931f-c95c641e5f02' on prisma.findUniquewebsite. Provided UUID, expected Int.
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at Document.validate (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:28989:20)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at serializationFn (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:31900:19)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at runInChildSpan (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:25102:12)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at PrismaClient._executeRequest (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:31907:31)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at consumer (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:31834:23)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at /home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:31839:51
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at /home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:31839:29
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at runInChildSpan (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:25102:12)
Oct 25 12:29:46 localhost nvm-exec[1498080]:     at PrismaClient._request (/home/love4taylor/applications/umami/node_modules/@prisma/client/runtime/index.js:31836:22) {
Oct 25 12:29:46 localhost nvm-exec[1498080]:   clientVersion: '4.5.0'
Oct 25 12:29:46 localhost nvm-exec[1498080]: }
```

**Comment by rohandebsarkar** [comment]
Encountered the same error.

**Comment by nicolasluckie** [comment]
Possible solution: https://github.com/umami-software/umami/issues/1583#issuecomment-1289988255

**Comment by briancao** [comment]
Migrations likely failed. The commands used [here ](https://github.com/umami-software/umami/issues/1583#issuecomment-1290006905) could be of use.

> I just ran 
`CREATE EXTENSION IF NOT EXISTS pgcrypto;` 
`delete from _prisma_migrations where migration_name = '04_add_uuid';`
> 
> And restarted Umami to re-apply.

**Comment by rohandebsarkar** [comment]
@briancao in my case, the migrations didn't fail. Still, I got that error.

Anyway, I applied the migrations manually, still the error persists. I am attaching the logs in case it helps.

```
022-10-25T06:19:48.600Z	4d9c033a-72ed-4fde-acda-5817ca1836cd	ERROR	PrismaClientValidationError: Argument id: Got invalid value '2c0d4795-5e6f-45b7-a2df-6c1096ea86de' on prisma.findUniquewebsite. Provided UUID, expected Int.    
at Document.validate (/var/task/node_modules/@prisma/client/runtime/index.js:28989:20)    
at serializationFn (/var/task/node_modules/@prisma/client/runtime/index.js:31900:19)    
at runInChildSpan (/var/task/node_modules/@prisma/client/runtime/index.js:25102:12)    
at PrismaClient._executeRequest (/var/task/node_modules/@prisma/client/runtime/index.js:31907:31)    
at consumer (/var/task/node_modules/@prisma/client/runtime/index.js:31834:23)    
at /var/task/node_modules/@prisma/client/runtime/index.js:31839:51     
at AsyncResource.runInAsyncScope (node:async_hooks:202:9)    
at /var/task/node_modules/@prisma/client/runtime/index.js:31839:29    
at runInChildSpan (/var/task/node_modules/@prisma/client/runtime/index.js:25102:12)    
at PrismaClient._request (/var/task/node_modules/@prisma/client/runtime/index.js:31836:22) {  clientVersion: '4.5.0'}2022-10-25T06:19:48.601Z	4d9c033a-72ed-4fde-acda-5817ca1836cd   ERROR	PrismaClientValidationError: Argument id: Got invalid value '2c0d4795-5e6f-45b7-a2df-6c1096ea86de' on prisma.findUniquewebsite. Provided UUID, expected Int.    
at Document.validate (/var/task/node_modules/@prisma/client/runtime/index.js:28989:20)    
at serializationFn (/var/task/node_modules/@prisma/client/runtime/index.js:31900:19)    
at runInChildSpan (/var/task/node_modules/@prisma/client/runtime/index.js:25102:12)    
at PrismaClient._executeRequest (/var/task/node_modules/@prisma/client/runtime/index.js:31907:31)    
at consumer (/var/task/node_modules/@prisma/client/runtime/index.js:31834:23)    
at /var/task/node_modules/@prisma/client/runtime/index.js:31839:51    
at AsyncResource.runInAsyncScope (node:async_hooks:202:9)    at /var/task/node_modules/@prisma/client/runtime/index.js:31839:29    
at runInChildSpan (/var/task/node_modules/@prisma/client/runtime/index.js:25102:12)    
at PrismaClient._request (/var/task/node_modules/@prisma/client/runtime/index.js:31836:22) {  clientVersion: '4.5.0'}RequestId: 4d9c033a-72ed-4fde-acda-5817ca1836cd Error: Runtime exited with error: exit status 1Runtime.ExitError
```

**Comment by briancao** [comment]
@rohandebsarkar Because you manually applied the migration, you'll likely need to rebuild umami or run `yarn build-db` to rebuild the latest version of the prisma client.

**Comment by rohandebsarkar** [comment]
> @rohandebsarkar Because you manually applied the migration, you'll likely need to rebuild umami or run `yarn build-db` to rebuild the latest version of the prisma client.

Thank you for the reply. I tried rebuilding umami on Vercel, also tried `yarn build-db` locally and hosted locally. But the issue is still there. I can not see any statistics on the dashboard as well as on the website page.

**Comment by briancao** [comment]
Ok, might be a legitimate non-migration related bug.  Let me look into it.

**Comment by rohandebsarkar** [comment]
Commit ebf439c5987a8e7480df6af81bc33907832c0b6b seems to fix the problem, thanks for the quick fix.

**Comment by mikecao** [comment]
Fixed in v1.39.2.
